### PR TITLE
LINK-1449 | Signups require strong identification from registration users

### PR DESCRIPTION
**Pull request by harriris-vincit**
### Description
After these changes, the "suomifi" identification will be required for registration users (`RegistrationUserAccess`) to access signup data.
### Closes
[LINK-1449](https://helsinkisolutionoffice.atlassian.net/browse/LINK-1449)

[LINK-1449]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-1449?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by tuomas777 — APPROVED** [review]
LGTM 👍 Would be nice if the WIP commit was squashed before merging.

**Comment by harriris-vincit** [comment]
Now squashed the commits into just one. Will merge once the checks have passed.

**Pull request merged**
